### ISSUE TITLE
What is the learning rate that has been used during the fine-tuning of XLM(MLM+TLM) for XNLI task("Zero shot setting")? 

### ISSUE THREAD

**Issue by gourango01** [facebookresearch/XLM#56]
In the paper, Section 5.1,  "We sample the learning rate of the Adam optimizer with values from 5.10−4 to 2.10−4" but on the github it is "5.10-6" .

**Comment by aconneau** [comment]
We used the following learning rates: "adam,lr=0.00001", "adam,lr=0.000005", "adam,lr=0.000002"
It's a typo in the paper.
